docker/build-sudo-images.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+# Build fast OS docker images that run slower steps ahead of time, to save time running tests
+
+cd "$(dirname "$0")"
+sed -i .orig -e "s#@@MYUSERID@@#`id -u`#g" sudo-*/Dockerfile
+docker build -t sudo-debian:wheezy sudo-debian
+docker build -t sudo-centos:6 sudo-centos6
+docker build -t sudo-centos:7 sudo-centos7
+docker build -t sudo-opensuse:13.2 sudo-opensuse
+docker build -t sudo-ubuntu:14.04 sudo-ubuntu14
```

Comment:
12.04.5 LTS will be EOL at April 2017
12.04.4 LTS already is EOL

On 08.12.2015 19:19, Sam Van Oort wrote:
>
> In docker/build-sudo-images.sh 
> <https://github.com/jenkinsci/packaging/pull/25#discussion_r46991139>:
>
> > @@ -0,0 +1,12 @@
> > +#!/bin/bash
> > +
> > +# Build fast OS docker images that run slower steps ahead of time, to save time running tests
> > +
> > +cd "$(dirname "$0")"
> > +sed -i .orig -e "s#@@MYUSERID@@#`id -u`#g" sudo-*/Dockerfile
> > +docker build -t sudo-debian:wheezy sudo-debian
> > +docker build -t sudo-centos:6 sudo-centos6
> > +docker build -t sudo-centos:7 sudo-centos7
> > +docker build -t sudo-opensuse:13.2 sudo-opensuse
> > +docker build -t sudo-ubuntu:14.04 sudo-ubuntu14
>
> @rsandell <https://github.com/rsandell> Are you suggesting 12.04 
> should be one of the test images here? It's fairly easy to add a new 
> image (does take a bit of tweaking to ensure it behaves correctly 
> though). I suppose the question is: what's the smallest set of base 
> images we can test against to give reasonable coverage of service 
> behaviors?
>
> —
> Reply to this email directly or view it on GitHub 
> <https://github.com/jenkinsci/packaging/pull/25/files#r46991139>.
>

-- 


Bernhard Cygan

Senior Solution Architect

CloudBees

Berlin, Germany

CloudBees Logo

*Mobile:* +49 173 5723610

*E-Mail:* bcygan@cloudbees.com

*Skype:* b.cygan

*Google Hangout:* bcygan@cloudbees.com



Blog zum CJP Release v1511 / Blog about CJP release v1511 
<https://www.cloudbees.com/blog/cloudbees-jenkins-platform-1511-november-2015-general-availability>

Webinar zum CJP Release v1511 / Webinar for CJP release v1511 
<https://pages.cloudbees.com/2015_1201_Webinar_CloudBees_Jenkins_Platform_v1511_Landing-Page.html>



Unsere nächsten Veranstaltungen / Our next events:

OOP, München <http://www.oop-konferenz.de>